templates/newsletter.html

```diff
@@ -1,16 +1,18 @@
-<section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup n-newsletter-signup" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
+<section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup{{#if theme}} n-newsletter-signup--{{theme}}{{/if}}" data-newsletter-user-subscribed="{{userIsSubscribed}}" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
```

Comment:
How about `{{#if userIsSubscribed}}data-newsletter-user-subscribed{{/if}}`. Feels neater. Also `data-newsletter-user-is-subscribed` ?